utility/importer/brave_importer.cc

```diff
@@ -524,8 +522,9 @@ bool ParsePaymentsPreferences(BraveLedger* ledger,
       break;
 
     default:
-      payments->min_visit_time =
-        braveledger_ledger::_default_min_publisher_duration;
+      payments->min_visit_time = 8;
```

Comment:
I think a better fix here would be to have a different gn file for this dep and move the constant there.

Reply:
agree, just wasn't sure if it was worth it for this since it's just migration from browser-laptop. I can open a ticket if you want

Reply:
@bridiver yes please, issue is fine for now. Thanks!

Reply:
https://github.com/brave/brave-browser/issues/4359